### PR TITLE
Fix golint failures in cmd/kube-controller-manager

### PR DESCRIPTION
**Pull request by xiaojingchen**
**What type of PR is this?**

/kind cleanup

**What this PR does?**:

fix golint failures in cmd/kube-controller-manager

**Which issue(s) this PR fixes**:

Ref: #68026

**Does this PR introduce a user-facing change?**:

NONE

### CODE REVIEW

**Comment by draveness** [comment]
/ok-to-test
/priority backlog
/release-note-none

please rebase the master.

File: cmd/kube-controller-manager/app/controllermanager.go [hunk 1]
```diff
@@ -328,6 +332,7 @@ type ControllerContext struct {
 	ResyncPeriod func() time.Duration
 }
 
+// IsControllerEnabled check if the context's controllers enabled or not
```
Comment (SataQiu) [inline cmd/kube-controller-manager/app/controllermanager.go]: Small nits. Others lgtm.
```suggestion
// IsControllerEnabled checks if the context's controllers enabled or not
```

File: cmd/kube-controller-manager/app/import_known_versions.go [hunk 2]
```diff
@@ -14,12 +14,13 @@ See the License for the specific language governing permissions and
...
 */
 
+// Package app imports the API groups that the client will support
 // TODO: Remove this file when namespace controller and garbage collector
 // stops using legacyscheme.Registry.RESTMapper()
 package app
 
-// These imports are the API groups the client will support.
 import (
+	// These imports are the API groups the client will support.
```
Comment (sttts) [inline cmd/kube-controller-manager/app/import_known_versions.go]: The comment is probably wrong nowadays. But it was wrong before, so ok.

**Pull request merged**